index.html

```diff
@@ -306,9 +234,73 @@ <h1>CONTACT US</h1>
               <a href="#"><img src="./icons/em.png" alt="image" /></a>
             </li>
           </ul>
-          <span class="email">wuyepabdul@gmail.com</span>
```

Comment:
![port42](https://user-images.githubusercontent.com/55635977/130155616-63c501d8-6a2c-4310-b278-f5a1edbb3f7d.png)

- [ ] Kindly arrange the `About myself` heading text with the paragraph on the same line, heading should be on left and paragraph on right but in the same line

- [ ] You can also work on the `Let's connect` heading with the icons to align them under the about myself heading.